### ISSUE TITLE
librtmp acknowledgement and LGPL license included at project root?

### ISSUE THREAD

**Issue by REDspaceMHackett** [ant-media/LibRtmp-Client-for-Android#50]
IANAL, but the librtmp code that you have included in your library is under the [LGPL](http://www.gnu.org/copyleft/lgpl.html), and it looks like Section 5 says that one can only create a combined library if the derivative is also LGPL. At the very least, I would expect you need to declare that this library include code licensed under the LGPL, and give the librtmp copyright in your NOTICE.

I'm worried that it is misleading to potential users of this library to offer it as Apache-licensed, when the core code is not under that license. It could come back to bite clients in commercial environments.

**Comment by mekya** [comment]
Yes you are right. I can declare this library include code licensed under the LGPL and add librtmp copyright in NOTICE.

Btw, what should be the license of this library? Is it possible to help me in these issues.
